whitelist.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <config>
-    <items>
-        <port>3322</port>
-    </items>
+  <items>
+    <port>3322</port>
+    <IPAddress>localhost</IPAddress>
```

Comment:
we don't set ip as a config at it can be obtained from TCP request and then checked whether it belongs to the internal network (i.e. network mask)